app/editor_ui/mixins.py

```diff
@@ -139,3 +140,68 @@ def dispatch(self, request, *args, **kwargs):
             )
 
         return super().dispatch(request, *args, **kwargs)
+
+
+class BreadCrumbsMixin:
+    """
+    A mixin to help with the display of breadcrumbs via the TNA Breadcrumbs component
+
+    Usage:
+    - Add BreadCrumbsMixin to your view class
+    - Set the `breadcrumb` property in the view class
+
+    Note:
+    - The implicit assumption is that all view urls are suffixed with '/'. All urls for views must have the '/' suffix,
+    if the BreadCrumbsMixin is to work.
```

Comment:
Do you think this documentation is sufficient? Or does more info need to be added?